Added ignore numbers variable argument (#5)
neofinancial/ccscan: CHANGELOG.md

# Changelog

## 1.0.5 (March 5, 2020)

> - added new optional argument `excluded-numbers` which can take in an array of numbers you wish to exclude from the scan

Reviewer: ```suggestion
- Add `ignore-numbers` argument which takes an array of numbers you want to ignore
```
